tests/lib/store.sh

```diff
@@ -30,12 +30,23 @@ teardown_staging_store(){
 }
 
 init_fake_refreshes(){
-    local dir=$1
+    local dir="$1"
     shift
 
     fakestore make-refreshable --dir "$dir" "$@" 
 }
 
+make_snap_installable(){
+    local dir="$1"
+    local snap_path="$2"
+
+    cp -a "$snap_path" "$dir"
+    fakestore new-snap-declaration --dir "$dir" "${snap_path}"
+    snap ack $dir/asserts/*.snap-declaration
```

Comment:
I would prefer not to need the wildcard here "*"  , maybe the new-* commands can print the file they just created